src/geo_index.c

```diff
@@ -68,12 +68,12 @@ int GeoFilter_Parse(GeoFilter *gf, ArgsCursor *ac, QueryError *status) {
   }
 
   gf->unit = AC_GetStringNC(ac, NULL);
+  gf->unit = rm_strdup(gf->unit);
```

Comment:
do we need duplication here at all? makjes more sense to use a constant value here (i see this is not a new change, but if were going through this lets do it right)

Reply:
I think we do in case of cursors

Reply:
Not follow ..

Reply:
see #922